config/samples/psm/go_example_loadtest_proxied.yaml

```diff
@@ -0,0 +1,153 @@
+# Load test configurations generated from a template by loadtest_config.py.
+# See documentation below:
+# https://github.com/grpc/grpc/blob/master/tools/run_tests/performance/README.md#grpc-oss-benchmarks
+apiVersion: e2etest.grpc.io/v1
+kind: LoadTest
+metadata:
+  annotations:
+    scenario: go_generic_sync_streaming_ping_pong_secure
+    uniquifier: basic
+  labels:
+    language: go
+    prefix: examples
+  name: psm-go-proxied
+spec:
+  clients:
+  - build:
+      args:
+      - build
+      - -o
+      - /src/workspace/bin/worker
+      - ./benchmark/worker
+      command:
+      - go
+    clone:
+      gitRef: master
+      repo: https://github.com/grpc/grpc-go.git
+    language: go
+    name: '0'
+    run:
+    - args:
+      - -c
+      - |
+        timeout --kill-after="${KILL_AFTER}" "${POD_TIMEOUT}" \
+            /src/workspace/bin/worker --driver_port="${DRIVER_PORT}"
+      command:
+      - bash
+      env:
+      - name: GRPC_GO_LOG_VERBOSITY_LEVEL
+        value: "99"
+      - name:  GRPC_GO_LOG_SEVERITY_LEVEL
+        value: info
+      name: main
+    - args:
+      - -default-config-path 
+      - containers/runtime/xds/config/default_config.json 
+      - -path-to-bootstrap
+      - containers/runtime/xds/bootstrap.json
+      command:
+      - main
+      image: gcr.io/grpc-testing/runtime/xds:test
```

Comment:
Right, since they requires the images like the pre-built examples I think they should be templates . 
For the pefix, I agree with setting PSM_IMAGE_PREFIX=gcr.io/grpc-testing/e2etest/runtime/psm to emphasize psm test. I will update them to be template.